README.rst

```diff
@@ -1,15 +1,15 @@
 Python Client for AI Platform Notebooks
 =================================================
 
-|beta| |pypi| |versions|
+|ga| |pypi| |versions|
```

Comment:
The README file under docs still shows beta: https://github.com/googleapis/python-notebooks/blob/ga-release/docs/README.rst . Can you try the following?

```
rm -rf docs/README.rst
cd docs && ln -s ../README.rst README.rst
```

Reply:
Done, PTAL.